docs/source/bentocloud/how-tos/manage-deployments.rst

```diff
@@ -40,41 +42,83 @@ To retrieve details about a specific Deployment:
+        created_at: '2024-02-20 09:27:52'
+        created_by: bentoml-user
+        config:
+          envs: []
+          services:
+            Summarization:
+              instance_type: cpu.2
+              scaling:
+                min_replicas: 1
+                max_replicas: 2
+              envs: []
+              deployment_strategy: Recreate
+              extras: {}
+              config_overrides:
+                traffic:
+                  timeout: 10
+        status:
+          status: running
+          created_at: '2024-02-20 09:27:52'
+          updated_at: '2024-02-21 05:46:18'
+
   .. tab-item:: Python API
 
+    To get basic information of a Deployment:
+
     .. code-block:: python
 
-      bentoml.deployment.get(name="deployment-1")
-
-Expected output in YAML:
-
-.. code-block:: yaml
-
-    name: summarization
-    bento: summarization:ghfvclwp2kwm5e56
-    cluster: aws-ca-1
-    endpoint_url: https://summarization-test--aws-ca-1.mt1.bentoml.ai
-    admin_console: https://test.cloud.bentoml.com/deployments/summarization/access?cluster=aws-ca-1&namespace=test--aws-ca-1
-    created_at: '2024-02-20 09:27:52'
-    created_by: bentoml-user
-    config:
-      envs: []
-      services:
-        Summarization:
-          instance_type: cpu.2
-          scaling:
-            min_replicas: 1
-            max_replicas: 2
-          envs: []
-          deployment_strategy: Recreate
-          extras: {}
-          config_overrides:
-            traffic:
-              timeout: 10
-    status:
-      status: running
-      created_at: '2024-02-20 09:27:52'
-      updated_at: '2024-02-21 05:46:18'
+      import bentoml
+
+      dep = bentoml.deployment.get(name="deploy-1")
+      print(dep)
+      # Output: DeploymentInfo(name='deploy-1', admin_console='https://test.cloud.bentoml.com/deployments/deploy-1/access?cluster=aws-ca-1&namespace=test--aws-ca-1', created_at='2024-03-01 05:00:19', created_by='bentoml-user', cluster='aws-ca-1')
+
+      print(dep.name)  # Print the name of the Deployment
+      print(dep.admin_console)  # Print the URL to the admin console for this Deployment
+      print(dep.created_at)  # Print the creation time
+      print(dep.created_by)  # Print the user who created the Deployment
+      print(dep.cluster)  # Print the cluster where the Deployment is hosted
+
+    To check the Deployment's status:
+
+    .. code-block:: python
+
+      import bentoml
+
+      dep = bentoml.deployment.get(name="deploy-1")
+      status = dep.get_status()
+      print(status.to_dict()) # Show the current status of the Deployment
+      # Output: {'status': 'running', 'created_at': '2024-03-01 05:00:19', 'updated_at': '2024-03-06 03:55:17'}
+
+    .. note::
+
+       ``get_status()`` has a parameter ``refetch`` to automatically refresh the status, which defaults to ``True``. You can use ``dep.get_status(refetch=False)`` to disable it.
```

Comment:
do we need to include `get_bento()` and `get_endpoint_urls()` as well?

Reply:
Added `get_bento()`, but `get_endpoint_urls()` does not seem to be working. I used the following and I got an AttributeError: `'DeploymentInfo' object has no attribute 'get_endpoint_urls'`

```python
import bentoml
dep = bentoml.deployment.get(name="deploy-1")
urls = dep.get_endpoint_urls()
print(urls)
```

But I think this is fine as it was already documented here.
![Screenshot 2024-03-06 at 14 57 33](https://github.com/bentoml/BentoML/assets/65327072/b2001a5c-21e2-46e9-ae4c-63481bf896a9)